scripts/world_bank/wdi/output/WorldBank_StatisticalVariables.mcf

```diff
@@ -634,3 +634,163 @@ statType: dcs:measuredValue
 measuredProperty: dcs:amount
 transferType: dcs:OutwardRemittance
 
+
+Node: dcid:WorldBank/SH_DYN_MORT
```

Comment:
What's the reason for these additions across the file? See a couple of comments below where it seems that the SV DCIDs are getting renamed but the SV and its contents are the same. Any idea what happened here? We should keep the SV DCIDs as checked in under ``google3/third_party/datacommons/schema/stat_vars/manual_wdi_stat_vars.mcf``